notebooks/tfx_pipelines/guided_projects/tfx_guided_project_vertex.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Nit for more clarity: "You can run unit tests using the <code>python -m</code> and supplying the path to the test module."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Done!</p>